.github/workflows/main.yml

```
@@ -0,0 +1,61 @@
+name: HTTPStan
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - '**'
+
+jobs:
+  tests:
+    name: HTTPStan tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-10.15]
+        python-version: [3.7, 3.8]
+      fail-fast: false
+    steps:
+      - name: Checkout github
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install poetry
+        run: |
+          pip install -U pip
+          pip install "poetry==1.0.9"
+
+      - name: Build libraries used by Stan
+        run: |
+          make
+
+      - name: Install dependencies
+        run: |
+          # export test dependencies from pyproject.toml, install them
+          poetry export -f requirements.txt --dev -o requirements.txt
+          pip install -r requirements.txt
```

```suggestion
        # export test dependencies from pyproject.toml, install them
        run: poetry export -f requirements.txt --dev -o requirements.txt && pip install -r requirements.txt
```
The comment is a YAML comment and not part of the executed text